docker-compose.ldap.yml

```diff
@@ -65,6 +65,24 @@ services:
       - "./local.env"
     tty: true
   # ---------------------------------
+  # Celery beat must only be used in > 1.2.0 installs. If setting to a previous install, remove this worker
+  # ----------------------------------
+  celery_beat:
+    image: "companyname/nautobot-ldap"
+    entrypoint: "sh -c 'nautobot-server celery beat -l $$NAUTOBOT_LOG_LEVEL'"
+    healthcheck:
```

Comment:
there is no health check for beat at this time

Reply:
Maybe:  `restart: "unless-stopped"`   will be good too?